datasets/ncslgr/ncslgr.py

```diff
@@ -0,0 +1,151 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""NCSLGR: a small American Sign Language corpus annotated with non-manual features"""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+import re
+from dataclasses import dataclass
+
+from tqdm import tqdm
+
+import datasets
+
+
+_DESCRIPTION = """
+A small American Sign Language corpus annotated with non-manual features
+"""
+
+_CITATION = """\
+@misc{dataset:databases2007volumes,
+    title={Volumes 2--7},
+    author={Databases, NCSLGR},
+    year={2007},
+    publisher={American Sign Language Linguistic Research Project (Distributed on CD-ROM~…}
+}
+"""
+
+_URL_ANNOTATIONS = "http://asl.cs.depaul.edu/corpus/elanBUcorpus.zip"
+_URL_VIDEOS = "http://asl.cs.depaul.edu/corpus/video.zip"
+
+_HOMEPAGE = "https://www.bu.edu/asllrp/ncslgr.html"
+
+
+@dataclass
+class NCSLGRConfig(datasets.BuilderConfig):
+    """BuilderConfig for NCSLGR."""
+
+    videos: bool = True
+
+
+class NCSLGR(datasets.GeneratorBasedBuilder):
+    """NCSLGR: a small American Sign Language corpus annotated with non-manual features"""
+
+    VERSION = datasets.Version("0.7.0")
+
```

Comment:
```suggestion
    BUILDER_CONFIG_CLASS = NCSLGRConfig
```
We need to specify that there's a custom config here ;)